File: src/A100/Models/Product.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace A100.Models
{
    public class Product
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public string ImageUrl { get; set; }
        public string ProductSubCategory { get; set; }
    }
}

File: src/A100/ClientApp/src/app/product/product.ts
export interface IProduct {
  id: number;
  title: string;
  price: number;
  description: string;
  imageUrl: string;
}

export class Product implements IProduct {
  id: number;
  title: string;
  price: number;
  imageUrl: string;
  description: string;
  constructor(_title:string,_price:number) {
    this.title = _title;
    this.price = _price;
  }
}
